### PR TITLE
Show 'quick links' to taxonomies in the sidebar of listings

### PR DESCRIPTION
**Pull request by bobdenotter**
See screenshot: 

Screenshot 2020-11-18 at 18 19 40

### CODE REVIEW

File: templates/content/listing.html.twig [hunk 1]
```diff
@@ -75,36 +77,34 @@
...
-                                                {{ taxonomyValue }}
-                                            </option>
-                                        {% endfor %}
-                                        </optgroup>
-                                    {% endif %}
-                                {% endfor %}
-                            </select>
+                            {% for taxonomy in contentType.get('taxonomy') %}
+                                {% set taxonomyDefinition = config.get('taxonomies/' ~ taxonomy) %}
+                                {% if taxonomyDefinition.behaves_like != 'tags' %}
```
Comment (bobdenotter) [inline templates/content/listing.html.twig:84]:
```suggestion
                                {% if taxonomyDefinition and taxonomyDefinition.behaves_like != 'tags' %}
```

**Review by I-Valchev — CHANGES_REQUESTED** [review]
@bobdenotter Were you going to make this so that the `Clear filter` button is visible when filtering on the taxonomies?

**Comment by bobdenotter** [comment]
Ah, yes! I _am_ going to do that!

**Comment by bobdenotter** [comment]
Tweaked and rebased!

**Review by I-Valchev — APPROVED** [review]
Me likes! 😇

Let's roll with it.

**Pull request merged**